ui/components/component-library/popover/popover.tsx

```diff
@@ -105,20 +105,21 @@ export const Popover: PopoverComponent = React.forwardRef(
         borderColor={BorderColor.borderMuted}
         borderRadius={BorderRadius.LG}
         backgroundColor={BackgroundColor.backgroundDefault}
-        padding={4}
+        padding={0}
         role={role}
         className={classnames(
           'mm-popover',
           {
             'mm-popover--open': Boolean(isOpen),
             'mm-popover--reference-hidden': Boolean(referenceHidden),
           },
+          'menu__container',
           className,
         )}
         ref={ref || setPopperElement}
         {...attributes.popper}
         {...(props as BoxProps<C>)}
-        style={{ ...styles.popper, ...contentStyle, ...props.style }}
+        style={{ ...styles.popper, ...contentStyle, ...props.style}}
       >
         {children}
         {hasArrow && (
```

Comment:
Let's not make any changes to the component file. We should be able to make the necessary overrides in `ui/components/multichain/global-menu/global-menu.js`